examples/ShowRecorder.cpp

```diff
@@ -108,6 +122,17 @@ void ShowRecorder::Stop() {
  */
 void ShowRecorder::NewFrame(const ola::client::DMXMetadata &meta,
                             const ola::DmxBuffer &data) {
+  if (!m_channeltrigger.empty()) {
+    if (recordingStarted == false) {
+      if (meta.universe != triggervalues.at(0)) {
+        return;
+      } else if (data.Get(triggervalues.at(1)) != 255) {
```

Comment:
I would probably default this to being > 50%; consider someone using this with a desk with faders, you don't want to have to require the fader to be right at the top of the range. Although I appreciate it might make doing it as part of a programmed recording, they'd probably want > 0%, so it could be added as a configurable value in future.